plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.scss

```diff
@@ -1,3 +1,9 @@
+/* *
+ * Internal dependencies
+ */
+/* This is imported here to avoid build errors when importing the same file in multiple places. */
+@import "./components/modals/modals.scss";
```

Comment:
This seems like a hacky way, but if it's the only way to do it, then let's go with it.